compiler/qsc_partial_eval/src/lib.rs

```diff
@@ -1311,6 +1311,23 @@ impl<'a> PartialEvaluator<'a> {
         args_span: PackageSpan,        // For diagnostic purposes only.
         callee_expr_span: PackageSpan, // For diagnostic puprposes only.
     ) -> Result<Value, Error> {
+        // check qubits here!
```

Comment:
```suggestion
        // check qubits for liveness here!
```

would this be correct?

Reply:
huh, this comment got left behind from notes to myself, but now I might as well clean it up and leave it in place for posterity.